### DISCUSSION TITLE
Welcome to SDE-Sheet Discussions!

### DISCUSSION THREAD

**Discussion by shivangshandilya** [ShivangShandilya/SDE-Sheet#3] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.
  * Let's make learning in public FUN, everyone.

**Comment by shiivanng** [comment]
Hello Shivang, Is this repo entirely in Java??

**Reply by shivangshandilya** [reply]
Yes the optimized solutions of all the DSA ques in this repository are in Java
